lib/formatters/getChallengeById.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = function (data) {
+  var tags;
+
+  if (!data) {
```

Comment:
I thought postgres will always return an array. It may be empty, but it I dont think it will be a falsey?